### PR TITLE
Release v1.2.1

### PR DESCRIPTION
**Pull request by jhermsmeier**
**PR Checklist:**

- [x] Bump the version number in the `package.json`'s `version` property
- [x] Bump the version number in the `npm-shrinkwrap.json`'s `version` property
- [x] Add a new entry to `CHANGELOG.md` by running `make changelog`
- [x] Re-take `screenshot.png` so it displays the latest version in the bottom
right corner
- [x] Revise the `updates.semverRange` version in `package.json`

**Release Checklist:**

- [x] Manual regression testing
- [ ] Merge this
- [ ] Tag the commit on master
- [ ] Publish the draft release
- [x] Publish packages on Bintray
- [ ] Publish packages on S3
- [ ] Forum post
- [ ] Update Etcher website
- [ ] Mailing List update

### CODE REVIEW

**Comment by jhermsmeier** [comment]
@jviotti mind taking that high-res screenshot again? :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.2.1 - 2017-12-06
+
+### Fixes
+
+- Fix handling of temporary read/write errors
+- Don't send initial Mixpanel events before "Anonymous Tracking" settings are loaded.
+- Fix verification step reading from the cache
```
Comment (lurch) [inline CHANGELOG.md]: Huh? According to https://github.com/resin-io/etcher/commits/master this was a feature in v1.2.0 ??

Comment (jhermsmeier) [reply CHANGELOG.md]: That's because the release branch itself was only merged after; but that definitely didn't end up in v1.2.0

**Comment by jviotti** [comment]
screen shot 2017-12-06 at 1 52 31 pm

**Comment by jviotti** [comment]
@jhermsmeier Looks like you should get yourself a company brand new Retina MacBook Pro, for "screenshooting" purposes :P

**Comment by lurch** [comment]
...or you could just write a little ImageMagick script to programatically redraw the version number in the bottom-right corner of the current screenshot ;-)

**Comment by lurch** [comment]
>  Publish the draft release
Publish packages on S3

If we want to 'fix' #1893 with Etcher v1.2.1 (I don't think there's any reason not to?), then you just need to make this change locally
```diff
--- a/Makefile
+++ b/Makefile
@@ -423,6 +423,7 @@ TARGETS +=  \
        electron-installer-redhat \
        cli-installer-tar-gz
 PUBLISH_AWS_S3 += \
+       $(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(TARGET_ARCH_APPIMAGE).AppImage \
        $(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(PLATFORM)-$(TARGET_ARCH_APPIMAGE).zip \
        $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(PLATFORM)-$(TARGET_ARCH).tar.gz
 PUBLISH_BINTRAY_DEBIAN += \
```
before running `make publish-aws-s3` (and I'll make the corresponding change to #1896 ). And then remember to also add the unzipped AppImages to the draft GitHub release.

**Comment by jhermsmeier** [comment]
@lurch why would we need to publish the AppImages to S3 as well?

**Comment by lurch** [comment]
I don't think there's any _real_ reason, it just feels more consistent. And if we add the change to the Makefile, then future snapshot uploads of Etcher will also have the unzipped AppImages available (since the snapshot releases currently only go to S3)

**Comment by jviotti** [comment]
> ...or you could just write a little ImageMagick script to programatically redraw the version number in the bottom-right corner of the current screenshot ;-)

We could do this with a small Spectron script btw

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.2.1 - 2017-12-06
+
+### Fixes
+
+- Fix handling of temporary read/write errors
+- Don't send initial Mixpanel events before "Anonymous Tracking" settings are loaded.
```
Comment (jviotti) [inline CHANGELOG.md]: Can we remove the trailing period here?

Comment (jhermsmeier) [reply CHANGELOG.md]: Done

**Comment by lurch** [comment]
> We could do this with a small Spectron script btw

I wonder if Spectron supports emulating high-res Retina displays?

File: CHANGELOG.md (hunk 2 above)
Comment (lurch) [reply CHANGELOG.md]: Oh, I assumed that the CHANGELOG.md was _only_ auto-generated by `versionist`, and there was no scope for hand-editing it? If we are "allowed" to hand-edit it, then I'd suggest removing "Add a sudo-prompt upon launch on Linux-based systems." as that change actually got reverted when we removed the Linux usbboot support, and it never made it into a published version of Etcher.

**Pull request merged**